volttrontesting/utils/platformwrapper.py

```diff
@@ -861,6 +874,11 @@ def startup_platform(self, vip_address, auth_dict=None,
                 # self.dynamic_agent.vip.pubsub.subscribe('pubsub', '', subscribe_to_all).get()
 
             if bind_web_address:
+                # Now that we know we have web and we are using ssl then we
```

Comment:
shouldn't this be part of init and not start if we want to create the admin user just once. There are testcases that restart platform wrapper within a testcase

Reply:
This needs to be handled during the instance. However, cert_profile_2 now checks if root ca is already created to avoid this issue. create_web_admin handles the case of repeated requests to create a new web admin within the same instance already.